src/user-defined-types/aliases.md

```diff
@@ -23,6 +23,10 @@ type PlayerInventory = RwLock<Vec<Arc<RefCell<Item>>>>;
 
 <details>
 
-C programmers will recognize this as similar to a `typedef`.
+- A [newtype](tuple-structs.html) is often a better alternative since
+  it creates a distinct type. Prefer `struct InventoryCount(usize)` to
+  `type InventoryCount = usize`.
```

Comment:
```suggestion
- A [newtype](tuple-structs.html) is often a better alternative since it creates
  a distinct type. Prefer `struct InventoryCount(usize)` to
  `type InventoryCount = usize`.
```